### PR TITLE
Add boilerplate for income and assets pdf mapping and tests.

### PR DESCRIPTION
**Pull request by scottsdevelopment**
## Summary

- Add boilerplate code for income and assets PDF mapping.

## Related issue(s)

- department-of-veterans-affairs/va.gov-team#82562

## Testing done

- Created new unit tests and fixtures

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1268,8 +1255,6 @@ DEPENDENCIES
   shoulda-matchers
   shrine
   sidekiq (~> 7.2.0)
-  sidekiq-ent!
```
Comment (TaiWilkin) [inline Gemfile.lock:1271]: Make sure you don't merge any changes to Sidekiq enterprise/pro (see [here](https://depo-platform-documentation.scrollhelp.site/developer-docs/base-setup-vets-api),  step 3).

**Review by TaiWilkin — CHANGES_REQUESTED** [review]
The code is working well - just need to clean up your Gemfile.lock file. 
Since this hasn't been hooked up yet (I presume because it will be called in the new intake job, which doesn't exist yet), I added a comment on the issue for the intake job linking to this PR for future reference.

**Review by wayne-weibel — CHANGES_REQUESTED** [review]
I fixed the Gemfile.lock, but now see some failing tests.  Once those are fixed this should be ready

**Review by TaiWilkin — APPROVED** [review]
Tests are passing, and the Gemfile.lock issue is corrected.

**Review by wayne-weibel — APPROVED** [review]

**Review by ryan-mcneil — APPROVED** [review]
👍

**Pull request merged**